packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1538,6 +1540,27 @@ export class ContainerRuntime
 			headerData.allowTombstone = request.headers[AllowTombstoneRequestHeaderKey];
 		}
 
+		const dataStoreNodePath = `/${id}`;
```

Comment:
This code should be in the `getDataStore` code dataStores.ts. That code handles other cases like data store never existed, data store is tombstoned, etc. We should keep all such code in one place.

Reply:
Should be updated now, please take a look :)